src/main/java/org/geysermc/discordbot/dump_issues/PlatformVersionDumpIssueCheck.java

```diff
@@ -0,0 +1,126 @@
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/GeyserDiscordBot
+ */
+
+package org.geysermc.discordbot.dump_issues;
+
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import pw.chew.chewbotcca.util.RestClient;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class PlatformVersionDumpIssueCheck extends AbstractDumpIssueCheck {
+
+    @Override
+    public boolean compatiblePlatform(String platform) {
+        return !platform.equals("STANDALONE");
+    }
+
+    @NotNull
+    @Override
+    public List<String> checkIssues(JSONObject dump) throws JSONException {
+        List<String> problems = new ArrayList<>();
+
+        JSONObject bootstrapInfo = dump.getJSONObject("bootstrapInfo");
+        String platformName = bootstrapInfo.getString("platformName");
+        String platformVersion = bootstrapInfo.getString("platformVersion");
+        String softwareName = platformName;
+
+        boolean isLatest = switch (platformName) {
+            case "Paper" -> {
+                String[] versionParts = platformVersion.split("-");
+                yield isLatestFillBuild("paper", versionParts[0], Integer.parseInt(versionParts[1]));
+            }
+            case "Velocity" -> {
+                String[] versionParts = platformVersion.split("-b");
+                String build = versionParts[versionParts.length-1];
+
+                yield isLatestFillBuild(
+                        "velocity", platformVersion.split(" ")[0],
+                        Integer.parseInt(build.replace(")", ""))
+                );
+            }
+            case "fabric" -> {
+                softwareName = "Fabric Loader";
+                yield isLatestFabricLoader(platformVersion);
+            }
+            default -> true;
+        };
+
+        if (!isLatest) {
+            problems.add("- Your version of %s is out of date, please consider updating your server software.".formatted(softwareName));
+        }
+
+        if (platformName.equals("Spigot") || platformName.equals("CraftBukkit")) {
+            problems.add("- You are running Geyser on Spigot, please consider switching to Paper for a more optimal experience.");
+        }
```

Comment:
Can we move this to another check? Just isnt really todo with platform versions more just a recommendation